scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Split.scala

```diff
@@ -84,25 +86,25 @@ case class Split(
 
   def onlyFor(splitTag: SplitTag, ignore: Boolean = false): Split =
     if (isIgnored || ignore || isNeededFor(splitTag)) this
-    else copy(neededTags = neededTags + splitTag)(line = line)
+    else copy(neededTags = neededTags + splitTag)
 
   def activateFor(splitTag: SplitTag): Split =
     if (isIgnored || isActiveFor(splitTag)) this
-    else copy(activeTags = activeTags + splitTag)(line = line)
+    else copy(activeTags = activeTags + splitTag)
 
   def preActivateFor(splitTag: SplitTag): Split =
     if (isIgnored) this
     else
       copy(
         activeTags = activeTags + splitTag,
         neededTags = neededTags + splitTag
-      )(line = line)
+      )
 
   def preActivateFor(splitTag: Option[SplitTag]): Split =
     if (isIgnored) this else splitTag.fold(this)(preActivateFor)
 
-  def forThisLine(implicit line: sourcecode.Line): Split =
-    if (isIgnored) this else copy()(line = line)
+  def forThisLine(implicit fileLine: FileLine): Split =
+    if (isIgnored) this else copy()
```

Comment:
however, I'm not convinced that it's not necessary here. are you sure copy will not take the original value, instead of the new one?

Reply:
och, yeah that was my mistake